test/__snapshots__/modules-option.test.js.snap

```diff
@@ -9787,7 +9787,7 @@ Array [
     "./modules/tests-cases/issue-589/source.css",
     "body:before {
   content: '';
-  background: url(replaced_file_protocol_/webpack/public/path/img.png);
+  background: url(replaced_file_protocol_/webpack/public/path/09a1a1112c577c2794359715edfcb5ac.png);
```

Comment:
Why it was changed?

Reply:
@alexander-akait not sure why , got automatically changed while updating snapshots

Reply:
We need investigate it